nubis/files/confd/conf.d/duo.toml

```diff
@@ -10,4 +10,4 @@ uid = 0
 gid = 0
 mode = "0600"
 
-reload_cmd = "puppet apply /etc/duo/duo_sshd_config_runtime.pp"
+reload_cmd = "/etc/duo/duo.sh"
```

Comment:
Let's try and not put executables in /etc ?

```/usr/local/bin/nubis-duo-reload``` maybe ?

Reply:
Ah yeah I can fix that